README.md

```diff
@@ -1,13 +1,29 @@
 # MicroPython Workshop
 
-The purpose of this repository is to make a Sphinx-based web guide for running a MicroPython workshop. This is hosted on [Read the Docs](https://micropython-workshop.readthedocs.io/en/latest/).
+The purpose of this repository is to make a Sphinx-based web guide for running a
+MicroPython workshop. This is hosted on [Read the
+Docs](https://micropython-workshop.readthedocs.io/en/latest/).
 
-The purpose of the workshop will be to take someone that may be familiar with Python but has never used it on a microcontroller, and allow them to smoothly sail through the treacherous seas towards getting that first piece of code running that makes the hardware do something fun. From there it aims to provide the resources and guidance necessary to develop skills further and progress towards project-based learning.
+The workshop targets someone that may be familiar with Python but is new to
+MicroPython - or even embedded development entirely. The purpose is to gently
+guide them towards getting that first piece of code running that makes the
+hardware do something fun. From there it aims to provide the resources and
+guidance necessary to develop skills further and progress towards project-based
+learning.
 
 ## Installation
 
-To build the documentation locally, you will need [Sphinx](https://www.sphinx-doc.org). This is most easily retrieved via `pip install -r requirements.txt`. This should ideally be done in the virtual environment of your choice - although note that you may need to add generated folders to the `exclude_patterns` list in `conf.py` to avoid them being included and then complained about by Sphinx during build.
+[Sphinx](https://www.sphinx-doc.org) is used to build the documentation. To
+install using `virtualenv`:
```

Comment:
This is actually using `venv`, a similar-but-different module (`virtualenv` is legacy)

Reply:
Good point, I'll update that.